### ISSUE TITLE
access "keywords" field from DESCRIPTION

### ISSUE THREAD

**Issue by njtierney** [ropenscilabs/flipper#17]
I didn't know that this was a field! 

Check it out in @cboettig knitcitations [DESCRIPTION](https://github.com/cboettig/knitcitations/blob/master/DESCRIPTION#L30)

**Comment by mpadge** [comment]
That's a great idea. I'm taking a temporary break into the more bite-sized [`stellar`](https://github.com/ropenscilabs/stellar), for which I'll develop lots of stuff that can then enrich `flipper`

**Comment by njtierney** [comment]
Awesome! Also, `stellar` - great name! :)

**Comment by mpadge** [comment]
yeah, but check [here](https://github.com/ropenscilabs/stellar/issues/2), and please pipe in with any suggestions for alternative names!

**Comment by cboettig** [comment]
@njtierney @mpadge Yeah, so technically anything can be a field in DESCRIPTION (i.e. it's still a valid debian control file and the only complaint you'll see from R CMD CHECK is "unrecognized term, possible spelling error".  But the guidance we've gotten from CRAN is that we should explicitly namespace additional metadata terms, a la: https://github.com/ropensci/codemetar/blob/master/DESCRIPTION#L76

Of course you can access anything in DESCRIPTION using `read.dcf()` from base, but you might enjoy https://github.com/r-lib/desc instead.
